src/core/attributeformmodel.h

```diff
@@ -60,16 +62,31 @@ class AttributeFormModel : public QSortFilterProxyModel
     void setFeatureModel( FeatureModel *featureModel );
 
     bool constraintsValid() const;
-
+    /**
+     * Save the current (already existing) feature
+     */
     Q_INVOKABLE void save();
+    /**
+     * Create the current (not existing yet) feature
+     */
     Q_INVOKABLE void create();
+    /**
+     * Delete the current feature
+     */
+    Q_INVOKABLE void deleteFeature();
+    /**
+     * Get the attribute of the current feature by name
+     * \param name the name of the attribute
+     * \return value of the attribute
+     */
     Q_INVOKABLE QVariant attribute( const QString &name );
 
   signals:
     void featureModelChanged();
     void hasTabsChanged();
     void featureChanged();
     void constraintsValidChanged();
+    void setRelationFeatureId( QgsFeatureId featureId );
```

Comment:
Is this a signal or a setter? Is it maybe even unused (can't find it)

Reply:
unused.